ports/espressif/Makefile

```diff
@@ -120,19 +120,25 @@ CFLAGS += -DSTACK_CANARY_VALUE=0xa5a5a5a5
 
 #Debugging/Optimization
 ifeq ($(DEBUG), 1)
-  CFLAGS += -DDEBUG -ggdb
-  OPTIMIZATION_FLAGS ?= -Og
-  # You may want to enable these flags to make setting breakpoints easier.
-  # CFLAGS += -fno-inline -fno-ipa-sra
+	CFLAGS += -ggdb
+	ifeq ($(IDF_TARGET_ARCH),riscv)
+		OPTIMIZATION_FLAGS ?= -Os
+		CFLAGS += -DNDEBUG
+	else
+		OPTIMIZATION_FLAGS ?= -Og
+		CFLAGS += -DDEBUG
+	endif
+	# You may want to enable these flags to make setting breakpoints easier.
+	# CFLAGS += -fno-inline -fno-ipa-sra
 else
-  CFLAGS += -DNDEBUG -ggdb3
-  # RISC-V is larger than xtensa
-  # Use -Os for RISC-V when it overflows
-  ifeq ($(IDF_TARGET_ARCH),riscv)
-  OPTIMIZATION_FLAGS ?= -Os
-  else
-  OPTIMIZATION_FLAGS ?= -O2
-  endif
+	CFLAGS += -DNDEBUG -ggdb3
+	# RISC-V is larger than xtensa
+	# Use -Os for RISC-V when it overflows
+	ifeq ($(IDF_TARGET_ARCH),riscv)
+		OPTIMIZATION_FLAGS ?= -Os
+	else
+		OPTIMIZATION_FLAGS ?= -O2
+	endif
```

Comment:
The whitespace here is inconsistent: some tabs, some spaces. Doesn't matter that much, but might look different in different editors.

Reply:
It will look very annoying on *many* editors, and some will complain or replace it automatically across the whole document.
Why are we in the modern era not using 'soft tab: 4' for everything?

Reply:
> The whitespace here is inconsistent: some tabs, some spaces. Doesn't matter that much, but might look different in different editors.

This change makes it consistent with the rest of the file, which is tabs.

> Why are we in the modern era not using 'soft tab: 4' for everything?

This is [a Makefile which uses tabs explicitly for ".RECIPEPREFIX" by default](https://www.gnu.org/software/make/manual/html_node/Recipe-Syntax.html).

Reply:
OK, I could have sworn I was seeing spaces on some lines, but I don't see that now.